src/main/resources/db/migration/V7__ny_kolonne_angitt_antall.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE faktura_linje
+    ADD COLUMN antall NUMERIC(10, 2); -- maks antall: 999,999.99
```

Comment:
Lurer på om den er nødvendig. I tillegg, hvis den er nødvendig/ønskelig, burde det vel være nok med noen få tall. Dette er vel bare enhetsprisPerMåned * antall (artikler eller hva man kaller det)? Da burde det vel aldri gå over 10 elns (vanligvis max 3 mtp kvartal, men i de tilfelllene man oppretter mange fakturalinjer for første faktura kan den kanskje overstige 3?)

Reply:
Bra du tok det opp, mest sannsynlig blir det lagret max numeric 4 ##.##, _men hvem_ vet hva de ønsker å kunne "selge" per enhet i fremtiden :shrug: 

Gjør ingenting ved mindre du kan garantere at vi ikke blir å putte enhetspris på andre faktureringsprodukter eller lagre verdier høyere enn ##.## (o.l.) :+1:

Reply:
> Bra du tok det opp, mest sannsynlig blir det lagret max numeric 4 ##.##, 
Du har kommentert at maks antall er 999,999.99, så gikk utifra at det er det som er maxen på dette feltet.

> men hvem vet hva de ønsker å kunne "selge" per enhet i fremtiden
Man må jo se litt på hva faktureringskomponenten løser, og tenke seg hvor sannsynlig det er man trenger det i fremtiden. Hvis man først skal gjøre så mye på beregningen av "antall", så tror jeg vi fint kan utvide feltet senere også.

> Gjør ingenting ved mindre du kan garantere at vi ikke blir å putte enhetspris på andre faktureringsprodukter eller lagre verdier høyere enn ##.## (o.l.) 
Jeg kommer ikke på noen scenarier, hverken realistisk i praksis, eller fiktive eksempler hvor dette skulle ha behøvd mer enn ##.##, altså skjønner ikke hvordan man kan få antall til å bli 100.00 i noen perioder egentlig. Dette ville i så fall vært over 100 hele måneder. Altså at man sender én første faktura som bruker skulle ha betalt i over 8 år. 

> Vi trenger altså ikke å optimalisere databasen på dette nivået vil jeg tro for Melosys smile
Neida, men når du først legger til en constraint...

Reply:
Kanskje det er lettere å forklare hvorfor jeg ikke gjør endringer her:
Det er som å argumentere mot å bruke en generert `.gitignore` template som har ignore-linjer som ikke vi nødvendigvis trenger _i dag_. Det er bedre å bruke en template som er sikrere for fremtiden slik at vi slipper å komme tilbake til dette i fremtiden ved å måtte endre den.

Reply:
Man lager da en template basert på prosjektet man jobber i og sannsynligheten for at en ignore-linje vil være nødvendig. Har man en java-applikasjon, lager man template for det og tar ikke med ignore-linjer for andre programmeringsspråk, fordi vi _kan_  innblande litt Python i framtiden, selv om sjansen for det er minimal.

Med det argumentet burde ikke det eksistere noen constraints, fordi vi en gang i framtiden kan få muligheten til å støtte nye grenser, selv om funksjonaliteten er udefinert.

Reply:
Antalle "999,999.99" tilsvarer 83.333 år... tror det er relativt safe å si at man ikke skal fakturere for så mange måneder av gangen.

Reply:
Nettopp, så vi trenger ikke å endre på noe som helst. Det er ingen i fag som blir å se eller vite om at vi _kan_ støtte noen få tusen år :rofl: 

Jeg er villig til å redusere det til 6  hvis det føles bedre? ####.##

Reply:
Bruker ikke mer tid på dette